data/config/datasets/adaguc.tests.solarterminator.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<Configuration>
+  <Style name="soltstyle">
+    <Legend fixedclasses="true" tickinterval="0.1" tickround=".01">no2</Legend>
+    <Min>0</Min>
+    <Max>4</Max>
+  
+    <ShadeInterval min="0.00" max="1.00"    label="Night"    fillcolor="#000000"/>
+    <ShadeInterval min="1.00" max="2.00"    label="Astronomical Twilight"    fillcolor="#333333"/>
+    <ShadeInterval min="2.00" max="3.00"    label="Nautical Twilight"    fillcolor="#666666"/>
+    <ShadeInterval min="3.00" max="4.00"    label="Civil Twilight"    fillcolor="#999999"/>
+    <ShadeInterval min="4.00" max="5.00"    label="Day"    fillcolor="#FFFFFF"/>
+
+    <NameMapping name="shadedcontour"   title="Shaded categories" abstract="Drawing with rainbow colors, shading and contours"/>
+    <RenderMethod>shadedcontour</RenderMethod>
+
+  </Style>
+  
+  <Layer type="liveupdate">
+    <FilePath filter="" gfi_openall="true">{ADAGUC_PATH}data/datasets/solt.nc</FilePath>
```

Comment:
Is there still a file needed?

Reply:
Unfortunately, yes. I spent a day or two seeing what would happen without a fail, but the `CImageDataWriter` expects for there to be a file. In fact, as I wrote in the MR description, the legend takes the units from the file. I got a bit lost in the CImageWriter code to be honest (the addData part). 

Or maybe we can split the fileless functionality and do it as a separate MR, so that an "algorithmic" dataset can properly work with no files in the config. This is why I had to restore the default liveupdate behaviour that writes to a `CImage` directly without going through all the steps of the `CImageDataWriter`.